app/config/AppConfig.scala

```diff
@@ -20,6 +20,7 @@ import javax.inject.{Inject, Singleton}
 import play.api.Configuration
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
+//noinspection ScalaUnusedSymbol
```

Comment:
Can all of these `//noinspection ScalaUnusedSymbol` be removed please. The compiler doesn't warn about these, only IntelliJ's code analysis tool so we shouldn't be putting these in our code.

Reply:
Removed all no inspection comments as per review comment.